{"log":"I1003 05:25:08.920151       1 server.go:126] Version: v1.11.3\n","stream":"stderr","time":"2018-10-03T05:25:08.921074351Z"}
{"log":"W1003 05:25:08.926956       1 authorization.go:47] Authorization is disabled\n","stream":"stderr","time":"2018-10-03T05:25:08.92991723Z"}
{"log":"W1003 05:25:08.929652       1 authentication.go:55] Authentication is disabled\n","stream":"stderr","time":"2018-10-03T05:25:08.929935108Z"}
{"log":"I1003 05:25:08.929682       1 insecure_serving.go:47] Serving healthz insecurely on 127.0.0.1:10251\n","stream":"stderr","time":"2018-10-03T05:25:08.929939089Z"}
{"log":"E1003 05:25:08.939282       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1beta1.PodDisruptionBudget: Get https://172.31.29.133:6443/apis/policy/v1beta1/poddisruptionbudgets?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.939530136Z"}
{"log":"E1003 05:25:08.939817       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.PersistentVolumeClaim: Get https://172.31.29.133:6443/api/v1/persistentvolumeclaims?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.94003688Z"}
{"log":"E1003 05:25:08.939961       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.StorageClass: Get https://172.31.29.133:6443/apis/storage.k8s.io/v1/storageclasses?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.940047983Z"}
{"log":"E1003 05:25:08.940343       1 reflector.go:205] k8s.io/kubernetes/cmd/kube-scheduler/app/server.go:176: Failed to list *v1.Pod: Get https://172.31.29.133:6443/api/v1/pods?fieldSelector=status.phase%21%3DFailed%2Cstatus.phase%21%3DSucceeded\u0026limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.940869737Z"}
{"log":"E1003 05:25:08.940804       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.Node: Get https://172.31.29.133:6443/api/v1/nodes?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.940881762Z"}
{"log":"E1003 05:25:08.949632       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.Service: Get https://172.31.29.133:6443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.949880484Z"}
{"log":"E1003 05:25:08.953858       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.PersistentVolume: Get https://172.31.29.133:6443/api/v1/persistentvolumes?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.954062183Z"}
{"log":"E1003 05:25:08.953945       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1.ReplicationController: Get https://172.31.29.133:6443/api/v1/replicationcontrollers?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.954076574Z"}
{"log":"E1003 05:25:08.954423       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1beta1.ReplicaSet: Get https://172.31.29.133:6443/apis/extensions/v1beta1/replicasets?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.954756333Z"}
{"log":"E1003 05:25:08.957376       1 reflector.go:205] k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:130: Failed to list *v1beta1.StatefulSet: Get https://172.31.29.133:6443/apis/apps/v1beta1/statefulsets?limit=500\u0026resourceVersion=0: dial tcp 172.31.29.133:6443: connect: connection refused\n","stream":"stderr","time":"2018-10-03T05:25:08.957669099Z"}
{"log":"E1003 05:25:18.924902       1 reflector.go:205] k8s.io/kubernetes/cmd/kube-scheduler/app/server.go:176: Failed to list *v1.Pod: pods is forbidden: User \"system:kube-scheduler\" cannot list pods at the cluster scope\n","stream":"stderr","time":"2018-10-03T05:25:18.925119792Z"}
{"log":"I1003 05:25:19.832974       1 controller_utils.go:1025] Waiting for caches to sync for scheduler controller\n","stream":"stderr","time":"2018-10-03T05:25:19.833283831Z"}
{"log":"I1003 05:25:19.933291       1 controller_utils.go:1032] Caches are synced for scheduler controller\n","stream":"stderr","time":"2018-10-03T05:25:19.93373493Z"}
{"log":"I1003 05:25:19.934327       1 leaderelection.go:185] attempting to acquire leader lease  kube-system/kube-scheduler...\n","stream":"stderr","time":"2018-10-03T05:25:19.934649982Z"}
{"log":"I1003 05:25:36.493628       1 leaderelection.go:194] successfully acquired lease kube-system/kube-scheduler\n","stream":"stderr","time":"2018-10-03T05:25:36.49386891Z"}
